requirements-charm.txt

```diff
@@ -0,0 +1,14 @@
+# Required to serve assets with Gunicorn
```

Comment:
Don't we need changes to the renovate config file to track these too?

Reply:
You are right, we have to think what to do with renovate in this project... however as this is mostly a copy of the upstream project, it requires a bit more thinking. Do you mind if we let it for another PR (where we also have to see how to rebase with upstream and its tags.

Reply:
Sure